### ISSUE TITLE
rightupoutside event may not be supported?

### ISSUE THREAD

**Issue by KazuyaSerizawa** [michalochman/react-pixi-fiber#320]
### Description
Hi Micha, thanks for this amazing library!
I encountered an issue where `rightupoutside` event does not work well using this library.  Or maybe I'm doing something wrong. Either way, it would be great if you could help me figure this out.

### Steps to reproduce

1. I have a following code 
```
      const mouseDown = (data) => {
           ref.current.on('mousemove', () => {
                console.log('mousemove')
           });
           ref.current.on('mouseup', () => {
                console.log('mouseup')
           });
           ref.current.on('rightup', () => {
                console.log('rightup')
           });
           ref.current.on('mouseupoutside', () => {
                console.log('mouseupoutside')
           });
           ref.current.on('rightupoutside', () => {
                console.log('rightupoutside')
           });
      }
      <Container
        ref={ref}
        interactive={true}
        mousedown={mouseDown}
        rightdown={mouseDown}
      >
```
2. works fine everything except for `rightupoutside` event. 

### Additional info

- react-pixi-fiber version: 1.0.0-beta.13 (tried with v1.0.6 but didn't work either)
- React version: 17.0.2
- ReactDOM version: 17.0.2
- PixiJS version: 6.1.3

**Comment by michalochman** [comment]
Hi @KazuyaSerizawa, seems you are subscribing to this event using a reference to PixiJS object, bypassing this library. Is it working for you when using just PixiJS? I was never trying to use `rightupoutside` so I'm not sure it is even supported.

**Comment by KazuyaSerizawa** [comment]
@michalochman 
Thank you Michal! 
I did the further research and found that it was actually a PixiJS (at least with v6.1.3) issue as you say...

`rightupoutside` event does work when that event location is within the browser but doesn't when it's outside of the browser window, which I did not notice when I first posted this issue.

Thanks again for your reply and this amazing library. My team has a great respect for how you've been dealing with react-pixi library thing, which we all know it was originally forked from this repo. 

I think I'm closing this issue.
